CefSharp/ISelectClientCertificateCallback.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright © 2010-2016 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+using System;
+using System.Security.Cryptography.X509Certificates;
+
+namespace CefSharp
+{
+    /// <summary>
+    /// Callback interface used for fetching the selected certificate.
+    /// </summary>
+    public interface ISelectClientCertificateCallback : IDisposable
+    {
+        /// <summary>
+        /// Return the selected certificate
```

Comment:
`Return` seems out of place here (if it's copied from the `CEF` doco, then it should be improved).